### PR TITLE
NTP: Fix a couple of UI issues in the New Trade Protocol

### PR DESCRIPTION
**Pull request by ripcurlx**

### CODE REVIEW

File: core/src/main/java/bisq/core/presentation/SupportTicketsPresentation.java [hunk 1]
```diff
@@ -42,6 +43,11 @@
     @Getter
     private final BooleanProperty showOpenMediationTicketsNotification = new SimpleBooleanProperty();
 
+    @Getter
+    private final StringProperty numOpenRefundTickets = new SimpleStringProperty();
+    @Getter
+    private final BooleanProperty showOpenRefundTicketsNotification = new SimpleBooleanProperty();
```
Comment (lusarz) [inline core/src/main/java/bisq/core/presentation/SupportTicketsPresentation.java:49]: Are `numOpenRefundTickets` and `showOpenRefundTicketsNotification` variables used somewhere outside of `SupportTicketsPresentation` ?

**Review by sqrrm — APPROVED** [review]
utACK

File: core/src/main/java/bisq/core/presentation/SupportTicketsPresentation.java (hunk 1 above)
Comment (sqrrm) [reply core/src/main/java/bisq/core/presentation/SupportTicketsPresentation.java:49]: Not that I can see, just like the other properties.

Comment (ripcurlx) [reply core/src/main/java/bisq/core/presentation/SupportTicketsPresentation.java:49]: > Are `numOpenRefundTickets` and `showOpenRefundTicketsNotification` variables used somewhere outside of `SupportTicketsPresentation` ?

But yes, the getters for all properties are not necessary anymore (there was a use case in the past). I'll change it in one of my next PRs

Comment (lusarz) [reply core/src/main/java/bisq/core/presentation/SupportTicketsPresentation.java:49]: So why we need these properties ? Seems like we only set values, but never use them. Or I miss something ?

Comment (ripcurlx) [reply core/src/main/java/bisq/core/presentation/SupportTicketsPresentation.java:49]: > So why we need these properties ? Seems like we only set values, but never use them. Or I miss something ?

I changed it already locally where all except of ~one~ two properties (numOpenSupportTickets, showOpenSupportTicketsNotification) are removed

**Comment by chimp1984** [comment]
@ripcurlx @sqrrm  It seems support for RefundManager is missing. ArbitrationManager is for the current arbitrator, but RefundManager fro the NTP version of arbitration.

**Pull request merged**